### ISSUE TITLE
Plugin assumed legacy in PHPStorm

### ISSUE THREAD

**Issue by Kurre** [opus1269/smooth-scroller#1]
First of all, thank you for making this plugin.
I've been waiting for a good smooth scroller plugin (built-in 'smooth scrolling' option doesn't actually provide smooth scrolling..) for a long time :+1:

**Problem:**
Latest PHPStorm 2016.1 (145.256.26) doesn't load this plugin, due missing tags in plugin.xml

```
2016-03-11 08:46:08,822 [   4390]   INFO - llij.ide.plugins.PluginManager - Plugin 'Smooth Scroller' can't be loaded because: Plugin does not include any module dependency tags in its plugin.xml therefore is assumed legacy and can be loaded only in IntelliJ IDEA
```

https://github.com/opus1269/smooth-scroller/blob/master/resources/META-INF/plugin.xml#L54-L58

Is it commented out for a reason?

**Comment by opus1269** [comment]
No reason - just missed it. :smile: 

I released version 1.1.1, let me know if that fixes it.

**Comment by Kurre** [comment]
That fixed it, seems to be working nicely :+1:
